examples/research_projects/onnxruntime/text_to_image/train_text_to_image.py

```diff
@@ -412,6 +481,13 @@ def main():
 
     if args.gradient_checkpointing:
         unet.enable_gradient_checkpointing()
+        vae.enable_gradient_checkpointing()
+    else:
+        # Freeze vae if not training
+        vae.requires_grad_(False)
+
+    # Freeze text_encoder
+    text_encoder.requires_grad_(False)
```

Comment:
@patrickvonplaten Would this be a proper way in introducing gradient checkpointing for VAE in the training script?